src/Home/Home.module.scss

```diff
@@ -39,9 +39,42 @@
   color: white;
   margin-top: 5vh;
   text-align: center;
-  padding: 6vh;
+  padding: 19px 27px 43px 27px;
+  min-height: 16vh;
+  display: flex;
+  flex-flow: column;
+  justify-content: space-evenly;
+  align-items: center;
 }
 
 .descriptionText {
   text-align: left;
+  flex: 0.2 1;
+}
+
+@media screen and (min-width: 481px) {
+  .img {
+    background-position: center;
+  }
+  .description {
+    padding: 3vh 6vh 7vh;
```

Comment:
Why are you using `vh` here? Are you trying to make the padding responsive?

Reply:
As a matter of fact, I am :)

Reply:
👍